### PR TITLE
payRunId to payRunNumber, invoiceId to invoiceNumber

### PR DESCRIPTION
**Pull request by mugnivenko**
https://app.clickup.com/t/1wm5vnh

### CODE REVIEW

File: components/Pages/Payruns/PayrunList.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const PayrunList = ({ searchTerm, data }) => {
                 <Container display="flex" alignItems="center">
                   <Heading size="s">Payrun ID: </Heading>
```
Comment (akeamwoodsND) [inline components/Pages/Payruns/PayrunList.js:37]: need to change this text here, ask burak for what to change it to

Comment (akeamwoodsND) [reply components/Pages/Payruns/PayrunList.js:37]: Payrun Number
probably

File: components/Pages/Payruns/SinglePayRun/SinglePayRunOverview.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ export const SinglePayRunOverview = ({ payRunId, searchTerm, payRun, setInvoiceI
         <VerticalSeparator width="24px" />
         <Heading size="s">Invoice ID:</Heading>
```
Comment (akeamwoodsND) [inline components/Pages/Payruns/SinglePayRun/SinglePayRunOverview.js:62]: need to change heading here

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
We need to use the highlighted search on number

File: components/Pages/Payruns/SinglePayRun/SinglePayRunOverview.js (hunk 2 above)
Comment (mugnivenko) [reply components/Pages/Payruns/SinglePayRun/SinglePayRunOverview.js]: it's true

File: components/Pages/Payruns/PayrunList.js (hunk 1 above)
Comment (mugnivenko) [reply components/Pages/Payruns/PayrunList.js]: true

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
search is not working correctly
Screenshot 2021-12-06 at 13 59 30

**Pull request merged**